Simplenote/src/androidTest/java/com/automattic/simplenote/screenshots/ScreenshotTest.java

```diff
@@ -319,7 +327,7 @@ private void loadThemeSwitcherFromNotesList() {
         loadThemeSwitcherFromSettings();
     }
 
-    private void takeInterNoteLinkingScreenshotFromNotesList() throws InterruptedException {
+    private void takeInterNoteLinkingScreenshotFromNotesList(String noteName, String screenshotName) throws InterruptedException {
```

Comment:
I added these parameters to make the method seem less like a black box. In particular, I found it annoying to have to scroll or do some text search to find the place in which the screenshot file name was set.